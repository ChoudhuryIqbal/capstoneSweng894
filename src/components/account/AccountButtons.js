import React from 'react'
import { View } from 'react-native'
import { ButtonCustom } from '../common/ButtonCustom' 

/**
 * General customizable button 
 * 
 * @param {*} props 
 */
const AccountButtons = (props) => {
    return (
        <View>
            <ButtonCustom
                testID='AccountBtn'
                onPress={props.btnOne}
                buttonStyle={props.btnOneStyle}
                textStyle={props.txtOnetyle}
            >
                {props.txtOne}
            </ButtonCustom>
        </View>
    )
}

export default AccountButtons