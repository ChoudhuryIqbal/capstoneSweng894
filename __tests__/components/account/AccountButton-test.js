import React from 'react'
import AccountButtons from '../../../src/components/account/AccountButtons'
import { create } from 'react-test-renderer';

describe('AccountButtons correctly', () => {
    test('Matches the snapshot', () => {
        const button = create(<AccountButtons />);
        expect (button.toJSON()).toMatchSnapshot();
    })
  });