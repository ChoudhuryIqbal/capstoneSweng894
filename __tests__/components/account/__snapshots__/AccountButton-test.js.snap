// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AccountButtons correctly Matches the snapshot 1`] = `
<View>
  <View
    accessible={true}
    focusable={false}
    isTVSelectable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "opacity": 1,
      }
    }
  >
    <Text />
  </View>
</View>
`;
